src/content/subgames.cpp

```diff
@@ -37,7 +37,22 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 bool getGameMinetestConfig(const std::string &game_path, Settings &conf)
 {
 	std::string conf_path = game_path + DIR_DELIM + "minetest.conf";
-	return conf.readConfigFile(conf_path.c_str());
+
+	bool ret = conf.readConfigFile(conf_path.c_str());
+
+	for (const auto &name : conf.getNames()) {
+		if (name.compare(0, 7, "secure.") == 0) {
```

Comment:
one indentation level can be saved by using `if (name.compare(0, 7, "secure.") != 0) continue;` here